chpl/chpl-service/src/test/java/gov/healthit/chpl/scheduler/job/xmlgenerator/ContactXmlGeneratorTest.java

```diff
@@ -14,64 +14,62 @@
 import gov.healthit.chpl.domain.Contact;
 
 public class ContactXmlGeneratorTest {
-    
-    
+
     @Test
     public void addContactTest() throws XMLStreamException, IOException {
-        Contact contact = 
-                getContact(1l, "abc@def.com", "John", "Doe", "5555551212", "Mr.");
-        
+        Contact contact = getContact(1L, "abc@def.com", "John", "Doe", "5555551212", "Mr.");
+
         XMLOutputFactory factory = XMLOutputFactory.newInstance();
         StringWriter stringOut = new StringWriter();
         XMLStreamWriter writer = factory.createXMLStreamWriter(stringOut);
-        
+
         ContactXmlGenerator.addContact(contact, "contact", writer);
         writer.close();
-        
+
         String expected = "<contact><contactId>1</contactId><email>abc@def.com</email><firstName>John</firstName><lastName>Doe</lastName><phoneNumber>5555551212</phoneNumber><title>Mr.</title></contact>";
         assertEquals(expected, stringOut.toString());
         stringOut.close();
     }
-    
+
     @Test
     public void addContactTestWithANullValue() throws XMLStreamException, IOException {
-        Contact contact = 
-                getContact(1l, null, "John", "Doe", "5555551212", "Mr.");
-        
+        Contact contact = getContact(1L, null, "John", "Doe", "5555551212", "Mr.");
+
         XMLOutputFactory factory = XMLOutputFactory.newInstance();
         StringWriter stringOut = new StringWriter();
         XMLStreamWriter writer = factory.createXMLStreamWriter(stringOut);
-        
+
         ContactXmlGenerator.addContact(contact, "contact", writer);
         writer.close();
-        
+
         String expected = "<contact><contactId>1</contactId><firstName>John</firstName><lastName>Doe</lastName><phoneNumber>5555551212</phoneNumber><title>Mr.</title></contact>";
```

Comment:
This should no longer be the expected XML. Note that this means changes to the XSD!